### ISSUE TITLE
Compatibility of official caffe version

### ISSUE THREAD

**Issue by Yozey** [xipeng13/recurrent-face-alignment#3]
Hello Xipeng,

I am impressed by your work using RNN and encoder decoder network to solve the sequential face alignment problem. Really nice job! 

I would like to know if your code is compatible with officially released caffe version since it supports batch normalization as well. [https://github.com/BVLC/caffe/blob/master/include/caffe/layers/batch_norm_layer.hpp](BVLC-caffe-batch norm)

I found that the bn_param in your deploy is different from the official version and might need your modified version of caffe.

Thank u in advance and wish you all the best.

**Comment by Yozey** [comment]
Sorry for repeating issue No.1
